data/mods/Dark-Skies-Above/monsters/wild_aliens/strays.json

```diff
@@ -296,50 +453,347 @@
+    "type": "MONSTER",
+    "name": { "str": "crystal mass wall" },
+    "description": "A massive wall of thick, blocky crystals that glow faintly and crackle with electric energy.",
+    "default_faction": "stray",
+    "bodytype": "blob",
+    "categories": [ "ALIEN" ],
+    "species": [ "BIOCRYSTAL" ],
+    "volume": "680388 ml",
+    "weight": "680388 g",
+    "hp": 200,
+    "speed": 100,
+    "symbol": "^",
+    "color": "light_cyan_white",
+    "aggression": 5,
+    "morale": 100,
+    "melee_cut": 0,
+    "armor_bash": 10,
+    "armor_cut": 60,
+    "armor_stab": 60,
+    "vision_day": 5,
+    "vision_night": 5,
+    "luminance": 10,
+    "harvest": "exempt",
+    "special_when_hit": [ "ZAPBACK", 100 ],
+    "death_function": [ "DISINTEGRATE" ],
+    "flags": [ "IMMOBILE", "NOGIB", "ELECTRIC", "WARM", "NO_BREATHE", "NOHEAD", "ARTHROPOD_BLOOD" ]
+  },
+  {
+    "id": "dks_mon_crystal_hive",
+    "type": "MONSTER",
+    "name": { "str": "crystal mass hive" },
+    "description": "A towering mass of blue-purple crystal chunks coverd in small, fist-sized holes, from which drips a disgusting, faintly glowing rock slurry - perhaps a nutrient mix for its young.  After all, it seems to subtly pulsate, as if more organic than its mineral appearance would suggest.  The air around it perpetually crackles with palpable energy and menaces with a set of thick, meaty tendrils with razor-like barbs, with something else just as unseemly writhing just beneath the murky surface of the glassy 'rocks'.",
+    "default_faction": "stray",
+    "bodytype": "blob",
+    "categories": [ "ALIEN" ],
+    "species": [ "BIOCRYSTAL" ],
+    "volume": "875000 ml",
+    "weight": "200 kg",
+    "hp": 275,
+    "speed": 100,
+    "symbol": "*",
+    "color": "cyan",
+    "aggression": 5,
+    "morale": 100,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 8,
+    "melee_cut": 5,
+    "armor_bash": 10,
+    "armor_cut": 45,
+    "armor_stab": 45,
+    "vision_day": 5,
+    "vision_night": 5,
+    "harvest": "exempt",
+    "reproduction": { "baby_monster": "dks_mon_crystal_mite", "baby_count": 4, "baby_timer": 60 },
+    "special_attacks": [ [ "BOOMER_GLOW", 20 ], [ "STRETCH_ATTACK", 20 ], [ "LONGSWIPE", 20 ] ],
+    "special_when_hit": [ "ZAPBACK", 100 ],
+    "death_function": [ "BOOMER_GLOW", "DISINTEGRATE" ],
+    "flags": [ "SEES", "IMMOBILE", "NOGIB", "ELECTRIC", "BLEED", "WARM", "NO_BREATHE", "NOHEAD", "ARTHROPOD_BLOOD" ]
+  },
+  {
+    "id": "dks_mon_crystal_mite",
+    "type": "MONSTER",
+    "name": { "str": "crystal mite" },
+    "description": "A tiny, multilegged creature that appears to be made of a chunk of crystal. It skitters around on wire-like legs, eating bits of organic leftovers to gain mass in hopes of one day seeding a crystal colony of its own.",
+    "default_faction": "stray",
+    "bodytype": "spider",
+    "categories": [ "ALIEN" ],
+    "species": [ "BIOCRYSTAL" ],
+    "volume": "96 ml",
+    "weight": "96 g",
```

Comment:
I don't believe biocrystal has the density of water. 😛

Reply:
I admit to just nabbing the listing for weight/volume we have for chipmunks because that's roughly how big these little guys are and forgetting about it! I'll multiply weight by a factor of about ~1.20 which should bring us to about in line to the other densities.